doc/changelog/10-standard-library/14736-Arith2.rst

```diff
@@ -0,0 +1,12 @@
+- **Deprecated:**
+  some obsolete files from the ``Arith`` part of the standard library (``Div2``, ``Even``, ``Gt``, ``Le``, ``Lt``, ``Max``, ``Min``, ``Minus``, ``Mult``, ``NPeano``, ``Plus``).
+  ``Arith_prebase`` has been introduced temporarily to ensure compatibility, but it will be removed at the end of the
+  deprecation phase, e.g. in 8.17. Usage is thus discouraged.
+  All ``Hint`` declarations in the ``arith`` database have been moved to ``Arith_prebase`` and
+  ``Arith_base``.  To use the results about Peano arithmetic, we recommend importing
+  ``PeanoNat`` (or ``Arith_base`` to base it on the ``arith`` hint database) and using the ``Nat`` module.
+  To avoid deprecation warnings, ``Arith_base`` should be used in replacement of the deprecated files,
+  and calls to the statements from deprecated files should be replaced by the corresponding statements in ``PeanoNat.Nat``
+  suggested by the warning messages.
```

Comment:
That's better.  How about this if I've not changed the meaning?

(Also I moved what you **_should_** do before what you **_shouldn't_** do)

```suggestion
  Import ``Arith_base`` instead of these files.  References to items in the deprecated files should be replaced
  with references to `PeanoNat` as suggested by the warning messages.
  All ``Hint`` declarations in the ``arith`` database have been moved to ``Arith_prebase`` and
  ``Arith_base``.  To use the results about Peano arithmetic, we recommend importing
  ``PeanoNat`` (or ``Arith_base`` to base it on the ``arith`` hint database) and using the ``Nat`` module.
  ``Arith_prebase`` has been introduced temporarily to ensure compatibility, but it will be removed at the end of the
  deprecation phase, e.g. in 8.17.  Its use is thus discouraged.```